diff -urN ogle-0.8.0.org/configure.in ogle-0.8.0/configure.in
--- ogle-0.8.0.org/configure.in	Sun Oct 21 12:07:16 2001
+++ ogle-0.8.0/configure.in	Sun Oct 21 12:34:05 2001
@@ -7,9 +7,10 @@
 
 dnl Checks for programs.
 AC_PROG_CC
+AM_PROG_AS
 
 dnl AM_DISABLE_SHARED
-AM_PROG_LIBTOOL
+AC_PROG_LIBTOOL
 
 dnl Checks for typedefs, structures, and compiler characteristics.
 AC_C_BIGENDIAN
@@ -120,7 +121,7 @@
     [ AC_DEFINE([HAVE_CLOCK_GETTIME], 1, [ ])
       LIBS="$LIBS -lrt"
     ],
-    dnl AC_MSG_WARN(You need clock_gettime)
+    AC_MSG_WARN(You need clock_gettime)
   )
 )
 
